WaarpR66/src/main/java/org/waarp/openr66/client/utils/OutputFormat.java

```diff
@@ -246,7 +283,9 @@ public String toString(OUTPUTFORMAT format) {
               next = true;
             }
             builderKeys.append(entry.getKey());
-            builderValues.append(entry.getValue());
+            String value = entry.getValue().toString();
+            value = value.replace(';', ',');
+            builderValues.append(value);
```

Comment:
This handles the `;` case at the cost of modifying data, but does not address other cases like `\n`.
A [common way](https://en.wikipedia.org/wiki/Comma-separated_values) to do that is to surround values with `"`. 
This is supported by Excel, LibreOffice Calc, etc.

This seems more resilient :
```java
value = value.replace('"', '""');
builderValues.append('"').append(value).append('"');
```

Reply:
First doest ot work (single character, not string).
Second would imply that all value will be surrounded by `"`, which is not potentially what we want (number are not String).
So I propose:
- common clean = `\n` or `\r` replaced by ` ` 
- if there is a `;`, then surround by `"` as you requested (so only if needed)

Reply:
See proposal and tests